app.js

```diff
@@ -53,9 +54,10 @@ var pages = {
  */
 var pageContent = {
   css       : fs.readFileSync( './public/main.css', 'utf8' ),
+  enhance   : fs.readFileSync( './public/enhance.js', 'utf8' ),
   hashes    : {
-    css : md5( fs.readFileSync( './public/main.css', 'utf8' ) ),
-    js  : md5( fs.readFileSync( './public/tooling.js', 'utf8' ) )
+    css     : revisions.styles,
+    js      : md5( fs.readFileSync( './public/tooling.js', 'utf8' ) )
   },
```

Comment:
As I understand correctly you want to remove the md5 hash set up and add the scripts to the `rev.json`  as well, right?